roles/rabbitmq/tasks/main.yml

```diff
@@ -44,5 +44,8 @@
 - name: correct rabbitmqadmin modes
   file: group=root owner=root mode=0755 path=/usr/local/bin/rabbitmqadmin
 
+- name: permit access to rabbitMQ server
+  ufw: rule=allow to_port={{ rabbitmq.port }} proto=tcp
```

Comment:
We already have a UFW rule that allows all connections to controllers from computes, when communicating over the "pnode" or private network.

https://github.com/blueboxgroup/ursula/blob/master/roles/common/tasks/ufw.yml#L15-L18

This should be sufficient, if you correctly define your undercloud CIDR.